docs/pressable.md

```diff
@@ -0,0 +1,177 @@
+---
+id: pressable
+title: Pressable
+---
+
+Pressable is a Core Component wrapper that can detect various types of interaction states on any of it's defined children.
+
+```jsx
+<Pressable onPress={onPressFunction}>
+  <Text>I'm pressable!</Text>
+</Pressable>
+```
+
+## Example
+
+```js
+import React, {useState} from 'react';
+import {Pressable, StyleSheet, Text, View} from 'react-native';
+
+export default (App = () => {
+  const [timesPressed, setTimesPressed] = useState(0);
+
+  let textLog = '';
+  if (timesPressed > 1) {
+    textLog = timesPressed + 'x onPress';
+  } else if (timesPressed > 0) {
+    textLog = 'onPress';
+  }
+
+  return (
+    <View>
+      <Pressable
+        onPress={() => {
+          setTimesPressed((current) => current + 1);
+        }}
+        style={({pressed}) => [
+          {
+            backgroundColor: pressed ? 'rgb(210, 230, 255)' : 'white',
+          },
+          styles.wrapperCustom,
+        ]}>
+        <Text style={styles.text}>Press Me</Text>

```

Comment:
Sorry, I was thinking that this example would show both the style and children function API. 

```suggestion
        {({pressed}) => (<Text style={styles.text}>{pressed ? 'Pressed!' : 'Press Me'}</Text>)}

```